stdlib/public/core/CTypes.swift

```diff
@@ -109,6 +109,10 @@ public typealias CLongDouble = Float80
 #else
 #error("CLongDouble needs to be defined for this FreeBSD architecture")
 #endif
+#elseif $Embedded
+#if arch(x86_64) || arch(i386)
+public typealias CLongDouble = Float80
```

Comment:
This shouldn't be enabled universally, and if we want a default, it should be disabled.  `long double` should be `fp64`.

Reply:
The default seems to be 80-bit extended precision:
> On the x86 architecture, most C compilers implement long double as the 80-bit extended precision type supported by x86 hardware (generally stored as 12 or 16 bytes to maintain data structure alignment), as specified in the C99 / C11 standards (IEC 60559 floating-point arithmetic (Annex F)).

https://en.wikipedia.org/wiki/Long_double#Implementations

Reply:
The BSDs, Darwin, and Windows do not enable FP80.  I think that even Android under emulation did not enable FP80.  The default is disabled IMO.

Reply:
Fair enough, updated now